examples/preview/README.md

```diff
@@ -12,7 +12,7 @@ Install the latest plugin for this project and [WPGraphQL](https://wordpress.org
 
 The framework expects a few environment variables. Create a file in the root of the example project `/.env.local`.
 
-```
+```bash
 # Base URL for WordPress
 NEXT_PUBLIC_WORDPRESS_URL=http://yourwpsite.com
```

Comment:
We should probably update this one too. Not blocking and could be a separate PR.

```suggestion
NEXT_PUBLIC_WORDPRESS_URL=https://example.com
```